list.js

```diff
@@ -0,0 +1,14 @@
+const Docker = require("dockerode");
+const ishmael = new Docker({socketPath: '/var/run/docker.sock'});
+
+ishmael.listContainers({all:true}, (err, containers) => {
+  containers.forEach((entry) => {
+    let container = ishmael.getContainer(entry.Id);
+    container.stop((err, data) => {
+      container.remove();
+    });
+  });
+});
+
+let now = Math.floor(new Date().getTime() / 1000)
+ishmael.pruneContainers({until: now})
```

Comment:
This is an example of global spindown that should take place when three signals outlined above are triggered.